tests/functional/create_subnets_test.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2018 Cable Television Laboratories, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from drb_python.http_exceptions import AuthorizationError, ConnectionError
+from drb_python.drb_exceptions import ActionError, AlreadyExists
+from drb_python.subnet import Subnet
+from drb_python.http_session import HttpSession
+
+
+class SubnetTest(unittest.TestCase):
+    """
+    Tests for functions located in SubnetHttps
+    """
+    def test_create_subnet(self):
+        login = {'username': 'rocketskates', 'password': 'r0cketsk8ts'}
+        subnet_object = {
+            'address': '10.197.111.0',
+            'broadcast_address': '10.197.111.255',
+            'default_lease': 7200,
+            'dn': 'cablelabs.com',
+            'dns': '8.8.8.8',
+            'listen_iface': 'eno1',
+            'max_lease': 7200,
+            'name': 'Management_SUBNET',
```

Comment:
If I am not mistaken, DRB uses the 'name' field as the unique identifier. We should be generating this name with something like uuid.uuid4() so we can execute multiple tests against the same DRB instance.

Reply:
I don't like test with variable data.  But we can do this if thats the norm.